eventOrganizer/Date.java

```diff
@@ -43,7 +69,22 @@ public int compareTo(Date otherDate) {
         return Integer.compare(this.day, otherDate.day);
     }
 
+
+    /**
+     * Checks if a given year is a leap year or not
+     *
+     * @param year The int year to check for leap year status.
+     * @return True if the year is a leap year; otherwise, false.
+     */
     private boolean isLeapYear(int year){
+        /**
```

Comment:
I do not think that there should be a javadoc mid method. If I am wrong let me know but I think you mean for this to just be a comment